ui/components/app/edit-gas-fee-popover/edit-gas-tooltip/edit-gas-tooltip.js

```diff
@@ -0,0 +1,145 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { PRIORITY_LEVELS } from '../../../../../shared/constants/gas';
+import {
+  COLORS,
+  FONT_WEIGHT,
+} from '../../../../helpers/constants/design-system';
+import Typography from '../../../ui/typography';
+
+const EditGasToolTip = ({ priorityLevel, gasContext, t }) => {
```

Comment:
Do you need to pass `t` here? I think we can get this from either the i18n context or using Jyoti's new `I18nValue` component, right?

Reply:
So this was driving me crazy yesterday. For some reason the context is not accessible in this component? It seems it must have something to do with the `react-tippy` package we're using for this tool-tip. I'll try using Jyoti's I18nValue component, but I might need some help digging into why I can only get it working this way...

Reply:
Ah interesting. I might not have time to help today, but I'll see if I can dive into this tomorrow. Or @jpuri have you seen this before?

Reply:
<img width="314" alt="Screen Shot 2021-11-23 at 12 40 57 PM" src="https://user-images.githubusercontent.com/34557516/143084537-b181c4bd-15f5-41a3-b3ea-532c0470f1c4.png">
so here is what I'm seeing ('maxBaseFee' => '[maxBaseFee]') when I either use the `I18nValue` component or the `useI18nContext` hook directly...

Reply:
Yep `I18nValue` does not works inside tooltip I guess some context issue, but it should be possible to get `t` from i18nContext here

Reply:
Also fails :(